src/routes/console/account/mfa.svelte

```diff
@@ -28,6 +29,10 @@
             await sdk.forConsole.account.updateMfaAuthenticator(AuthenticatorType.Totp, code);
             await Promise.all([invalidate(Dependencies.ACCOUNT), invalidate(Dependencies.FACTORS)]);
             showSetup = false;
+            addNotification({
+                type: 'success',
+                message: 'Authenticator app added successfully'
```

Comment:
not sure if we should use `added` or `connected` here.
The status `Pill` component below uses `connected` though.

Reply:
@sara-k-48 done.